UDV-Core/examples/MAM/MAM.js

```diff
@@ -0,0 +1,316 @@
+//Working
```

Comment:
Please add a comment stating that this code is inspired from itowns examples with a lin kto the corresponding example.